README.md

```diff
@@ -309,6 +317,11 @@ import { NEOVIS_DEFAULT_CONFIG } from 'neovis.js';
     "caption": "name",
     "size": "pagerank",
     "community": "community",
+    //"image": 'https://visjs.org/images/visjs_logo.png',
```

Comment:
Why is that commented?

Reply:
I feel some people would think `image` is a mandatory field if we do not comment it out.